share/completions/sops.fish

```diff
@@ -0,0 +1,147 @@
+complete -x -c sops -n __fish_is_first_token -l encrypted-suffix -d "override the encrypted key suffix. When empty, all keys will be encrypted, unless otherwise marked with unencrypted-suffix"
+complete -x -c sops -n __fish_is_first_token -l unencrypted-regex -d "set the unencrypted key suffix. When specified, only keys matching the regex will be left unencrypted"
+complete -x -c sops -n __fish_is_first_token -l encrypted-regex -d "set the encrypted key suffix. When specified, only keys matching the regex will be encrypted"
+complete -r -c sops -n __fish_is_first_token -l config -d "path to sops' config file. If set, sops will not search for the config file recursively"
+complete -x -c sops -n __fish_is_first_token -l encryption-context -d "comma separated list of KMS encryption context key:value pairs"
+complete -x -c sops -n __fish_is_first_token -l set -d "set a specific key or branch in the input document. Value must be a json encoded string. (edit mode only). eg. --set '[\"somekey\"][0] {\"somevalue\":true}'"
+complete -x -c sops -n __fish_is_first_token -l shamir-secret-sharing-threshold -x -d "the number of master keys required to retrieve the data key with shamir (default: 0)"
+complete -r -c sops -n __fish_is_first_token -l output -d "save the output after encryption or decryption to the file specified"
+complete -x -c sops -n __fish_is_first_token -l keyservice -d "specify the key services to use in addition to the local one. Can be specified more than once. Syntax: protocol://address. Example: tcp://myserver.com:5000"
+
+# Global flags
+complete -F -c sops -s h -l help -d "show help"
+
+# exec-env flags
+complete -F -c sops -n "__fish_sops_commands exec-env" -l background -d "background the process and don\'t wait for it to complete"
+complete -x -c sops -n "__fish_sops_commands exec-env" -l user -a "(__fish_print_users)" -d "the user to run the command as"
+complete -F -c sops -n "__fish_sops_commands exec-env" -l enable-local-keyservice -d "use local key service"
+complete -x -c sops -n "__fish_sops_commands exec-env" -l keyservice -d "specify the key services to use in addition to the local one. Can be specified more than once. Syntax: protocol://address. Example: tcp://myserver.com:5000"
+
+# exec-file flags
+complete -F -c sops -n "__fish_sops_commands exec-file" -l background -d "background the process and don\'t wait for it to complete"
+complete -F -c sops -n "__fish_sops_commands exec-file" -l no-fifo -d "use a regular file instead of a fifo to temporarily hold the decrypted contents"
+complete -x -c sops -n "__fish_sops_commands exec-file" -l user -a "(__fish_print_users)" -d "the user to run the command as"
+complete -x -c sops -n "__fish_sops_commands exec-file" -l input-type -a "json;yaml;dotenv;binary" -d "currently json, yaml, dotenv and binary are supported. If not set, sops will use the file's extension to determine the output format"
```

Comment:
I'm surprised `-a "json;yaml;dotenv;binary"` works, we usually use spaces instead of ;

```suggestion
complete -x -c sops -n "__fish_sops_commands exec-file" -l input-type -a "json yaml dotenv binary" -d "currently json, yaml, dotenv and binary are supported. If not set, sops will use the file's extension to determine the output format"
```